lms/static/scripts/frontend_apps/services/index.js

```diff
@@ -1,6 +1,8 @@
 import { createContext, createElement } from 'preact';
 import { useContext, useMemo } from 'preact/hooks';
 
+export { GradingService } from './grading';
```

Comment:
Can you help me understand why this import is necessary?

Reply:
This is just a re-export for convenience so other code can do `import { ServiceName, useServices } from './services'`. I'm not strongly attached to whether we do this or not.